login-workflow/src/new-architecture/screens/VerifyCodeScreen/VerifyCodeScreen.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useState } from 'react';
+import { VerifyCodeScreenBase } from './VerifyCodeScreenBase';
+import { VerifyCodeScreenProps } from './types';
+import { useLanguageLocale } from '../../hooks';
+import { useRegistrationContext, useRegistrationWorkflowContext } from '../../contexts';
+
+/**
+ * Component that renders a screen that prompts a user to enter the confirmation code
+ * that was sent to the email address that they used to register.
+ *
+ * @param codeValidator used to test the input for valid formatting
+ * @param onResend function to call when the user clicks the 'resend code' button
+ * @param resendInstructions text to display ahead of the resend link/button
+ * @param resendLabel label for the resend link/button
+ * @param initialValue code used to pre-populate the field
+ * @param updateCode get the update code when user click the send again link
+ *
+ * @category Component
+ */
+
+export const VerifyCodeScreen: React.FC<VerifyCodeScreenProps> = (props) => {
+    const { t } = useLanguageLocale();
+    const regWorkflow = useRegistrationWorkflowContext();
+    const { actions } = useRegistrationContext();
+    const { nextScreen, previousScreen, screenData } = regWorkflow;
+
+    const {
+        codeValidator = (code: string): boolean | string =>
+            code?.length > 0 ? true : t('bluiRegistration:SELF_REGISTRATION.VERIFY_EMAIL.CODE_VALIDATOR_ERROR'),
+        onResend,
```

Comment:
You need to call an action onResend here as default. It is throwing an error :
<img width="1792" alt="Screenshot 2023-06-30 at 12 00 31 PM" src="https://github.com/etn-ccis/blui-react-workflows/assets/120575281/ce7869ca-49af-418c-bd8a-8c7d1e3c9f86">

Reply:
Fixed. Added call an action to onResend props